lib/utils.js

```diff
@@ -81,6 +81,27 @@ function validate_metadata(object, name) {
 }
 
 /**
+ * Create base url for registry.
+ * @param {String} protocol
+ * @param {String} host
+ * @param {String} prefix
+ * @return {String} base registry url
+ */
+function get_base_url(protocol, host, prefix) {
```

Comment:
As @Meeeeow suggested, this name seems get the url from somewhere. Either `joinUrl`  or better naming you might propose. Also rename as `camelCase`, is not something we enforce with `eslint` but I'll change that soon.